op-node/rollup/derive/isthmus_upgrade_transactions_test.go

```diff
@@ -26,10 +52,63 @@ func TestIsthmusSourcesMatchSpec(t *testing.T) {
+	require.Nil(t, deployOperatorFeeVault.To())
+	require.Equal(t, uint64(500_000), deployOperatorFeeVault.Gas())
+	require.Equal(t, operatorFeeVaultDeploymentBytecode, deployOperatorFeeVault.Data())
+
+	updateL1BlockProxySender, updateL1BlockProxy := toDepositTxn(t, upgradeTxns[3])
+	require.Equal(t, updateL1BlockProxySender, common.Address{})
+	require.Equal(t, updateIsthmusL1BlockProxySource.SourceHash(), updateL1BlockProxy.SourceHash())
+	require.NotNil(t, updateL1BlockProxy.To())
+	require.Equal(t, *updateL1BlockProxy.To(), common.HexToAddress("0x4200000000000000000000000000000000000015"))
+	require.Equal(t, uint64(50_000), updateL1BlockProxy.Gas())
+	require.Equal(t, common.FromHex("3659cfe6000000000000000000000000ff256497d61dcd71a9e9ff43967c13fde1f72d12"), updateL1BlockProxy.Data())
+
+	updateGasPriceOracleSender, updateGasPriceOracle := toDepositTxn(t, upgradeTxns[4])
+	require.Equal(t, updateGasPriceOracleSender, common.Address{})
+	require.Equal(t, updateIsthmusGasPriceOracleSource.SourceHash(), updateGasPriceOracle.SourceHash())
+	require.NotNil(t, updateGasPriceOracle.To())
+	require.Equal(t, *updateGasPriceOracle.To(), common.HexToAddress("0x420000000000000000000000000000000000000F"))
+	require.Equal(t, uint64(50_000), updateGasPriceOracle.Gas())
+	require.Equal(t, common.FromHex("3659cfe600000000000000000000000093e57a196454cb919193fa9946f14943cf733845"), updateGasPriceOracle.Data())
+
+	updateOperatorFeeSender, updateOperatorFee := toDepositTxn(t, upgradeTxns[5])
+	require.Equal(t, updateOperatorFeeSender, common.Address{})
+	require.Equal(t, updateOperatorFeeVaultSource.SourceHash(), updateOperatorFee.SourceHash())
+	require.NotNil(t, updateOperatorFee.To())
+	require.Equal(t, *updateOperatorFee.To(), common.HexToAddress("0x420000000000000000000000000000000000001b"))
+	require.Equal(t, uint64(50_000), updateOperatorFee.Gas())
+	require.Equal(t, common.FromHex("3659cfe60000000000000000000000004fa2be8cd41504037f1838bce3bcc93bc68ff537"), updateOperatorFee.Data())
+
+	gpoSetIsthmusSender, gpoSetIsthmus := toDepositTxn(t, upgradeTxns[6])
+	require.Equal(t, gpoSetIsthmusSender, common.HexToAddress("0xDeaDDEaDDeAdDeAdDEAdDEaddeAddEAdDEAd0001"))
+	require.Equal(t, enableIsthmusSource.SourceHash(), gpoSetIsthmus.SourceHash())
+	require.NotNil(t, gpoSetIsthmus.To())
+	require.Equal(t, *gpoSetIsthmus.To(), common.HexToAddress("0x420000000000000000000000000000000000000F"))
+	require.Equal(t, uint64(90_000), gpoSetIsthmus.Gas())
+	require.Equal(t, common.FromHex("291b0383"), gpoSetIsthmus.Data())
 
-	deployBlockHashesSender, deployBlockHashesContract := toDepositTxn(t, upgradeTxns[0])
-	require.Equal(t, deployBlockHashesSender, predeploys.EIP2935ContractDeployer)
+	deployBlockHashesSender, deployBlockHashesContract := toDepositTxn(t, upgradeTxns[7])
+	require.Equal(t, deployBlockHashesSender, common.HexToAddress("0xE9f0662359Bb2c8111840eFFD73B9AFA77CbDE10"))
```

Comment:
I believe this address changed. And it's reverting a change to reference the address from the `predeploys` package directly.